pkg/kubectl/cmd/create_role.go

```diff
@@ -50,20 +53,30 @@ type CreateRoleOptions struct {
 	Verbs         []string
 	Resources     []schema.GroupVersionResource
 	ResourceNames []string
+
+	DryRun       bool
+	OutputFormat string
```

Comment:
At some point, I think we'll need to start wrapping this logic inside of the `PrintObject` function.  You don't need to mess with it here.